### PR TITLE
fix: LEAP-16: Add validation to contextual scrolling

### PR DESCRIPTION
**Pull request by juliosgarbi**
### PR fulfills these requirements
- [x] Commit message(s) and PR title follows the format `[fix|feat|ci|chore|doc]: TICKET-ID: Short description of change made` ex. `fix: DEV-XXXX: Removed inconsistent code usage causing intermittent errors`
- [ ] Tests for the changes have been added/updated (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [x] Best efforts were made to ensure docs/code are concise and coherent (checked for spelling/grammatical errors, commented out code, debug logs etc.)
- [x] Self-reviewed and ran all changes on a local instance (for bug fixes/features)

#### Change has impacts in these area(s)
_(check all that apply)_
- [ ] Product design
- [x] Frontend

### Describe the reason for change
It was missing some validations to contextual scrolling.

#### What does this fix?
As it was missing some validations to contextual scrolling, there were some blocks of code that was running unnecessarily

#### What is the new behavior?
_(if this is a breaking or feature change)_

#### What is the current behavior?
_(if this is a breaking or feature change)_

#### What libraries were added/updated?
_(list all with version changes)_

#### Does this change affect performance?
_(if so describe the impacts positive or negative)_

#### Does this change affect security?
_(if so describe the impacts positive or negative)_

#### What alternative approaches were there?
_(briefly list any if applicable)_

#### What feature flags were used to cover this change?
_(briefly list any if applicable)_

### Does this PR introduce a breaking change?
_(check only one)_
- [ ] Yes, and covered entirely by feature flag(s)
- [ ] Yes, and covered partially by feature flag(s)
- [ ] No
- [ ] Not sure (briefly explain the situation below)

### What level of testing was included in the change?
_(check all that apply)_
- [ ] e2e
- [ ] integration
- [ ] unit

### Which logical domain(s) does this change affect?
_(for bug fixes/features, be as precise as possible. ex. Authentication, Annotation History, Review Stream etc.)_

### CODE REVIEW

File: src/tags/object/Paragraphs/Phrases.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ export const Phrases = observer(({ item, playingId, activeRef, setIsInViewport }
   // this function is used to animate the reading line when user seek audio
   const setSeekAnimation = useCallback(
     (isSeeking) => {
-      if (!isFF(FF_LSDV_E_278)) return;
+      if ((!isFF(FF_LSDV_E_278) || !item.contextscroll)) return;
```
Comment (hlomzik) [inline src/tags/object/Paragraphs/Phrases.js:58]: extra brackets here

**Review by hlomzik — APPROVED** [review]

File: src/tags/object/Paragraphs/Phrases.js (hunk 1 above)
Comment (juliosgarbi) [reply src/tags/object/Paragraphs/Phrases.js]: Done @hlomzik thanks! :)

File: src/tags/object/Paragraphs/Phrases.js [hunk 2]
```diff
@@ -122,8 +122,8 @@ export const Phrases = observer(({ item, playingId, activeRef, setIsInViewport }
 
   // when user click on play/pause button, the useEffect will be triggered and pause or play the reading line animation
   useEffect(() => {
-    if (!isFF(FF_LSDV_E_278)) return;
-
+    if (!isFF(FF_LSDV_E_278) || !item.contextscroll) return;
+
```
Comment (bmartel) [inline src/tags/object/Paragraphs/Phrases.js:126]:
```suggestion

```

**Review by bmartel — APPROVED** [review]

**Pull request merged**